kubernetes/src/main/resources/kubernetes/basic_auth.csv

```diff
@@ -1,2 +1,3 @@
 guest,guest,1001
 p4ssw0rd,admin,1002
+pr0m3th3us,prometheus,1003
```

Comment:
Very minor: personal preference for including new line characters at end of files.